axlearn/common/trainer.py

```diff
@@ -141,6 +141,9 @@ class Config(Module.Config):
         # By default, only trace on host 0.
         start_trace_process_indices: Union[Literal["all"], Sequence[int]] = [0]
 
+        # Log the step time for each individual step
+        log_step_time: bool = False
```

Comment:
Maybe it'll be more general to do:

```suggestion
        # Interval to log average step time. If None, defaults to 100.
        log_step_time_every_n_steps: Optional[int] = None
```

Note that we default to None instead of 100 to avoid triggering a large number of golden config updates.